README.md

```diff
@@ -1,6 +1,6 @@
 # Mattermost Mobile
 
-- **Minimum Server versions:** Current ESR version (5.9)
+- **Minimum Server versions:** Current ESR version (5.19)
```

Comment:
@amyblais bumping ESR version

Reply:
@enahum I don't think we should do this yet until 16th. I was also adding it via my changelog PR https://github.com/mattermost/mattermost-mobile/pull/4033/files#diff-04c6e90faac2675aa89e2176d2eec7d8.

Reply:
I'll submit another PR to change this back then

Reply:
done